src/cmd1.cc

```diff
@@ -3182,14 +3061,6 @@ void move_player_aux(int dir, int do_pickup, int run, bool_ disarm)
 		oktomove = FALSE;
 	}
 
-	/* Don't step on known traps. */
-	else if (disarm && (c_ptr->info & (CAVE_TRDT)) && !(p_ptr->confused || p_ptr->stun || p_ptr->image))
```

Comment:
I'll see if I can find it, I don't even remember what it is at the moment. It might have other uses too.